scrapers/devianteNetwork.yml

```diff
@@ -0,0 +1,49 @@
+name: "Deviante Network"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - sexworking.com/video/
+      - prettydirtyteens.com/video/
+      - loveherass.com/video/
+      - eroticspice.com/video/
+      - forgivemefather.com/video/
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1/text()
+      Date:
+        selector: //h1/preceding-sibling::h2/text()
+        postProcess:
+          - parseDate: January 2, 2006
+      Details: //h2[text()="Description"]/following-sibling::div/p/text()
+      URL: //link[@rel="canonical"]/@href
+      Tags:
+        Name: //a[contains(@href,"/videos/tags")]/text()
+      Performers:
+        Name: //h2//a[contains(@href,"/model/")]/text()
+      Studio:
+        Name: 
+          selector: //link[@rel="canonical"]/@href
+          postProcess:
+            - replace: 
+              - regex: .*www\.
+                with: ""
+              - regex: \.com.+
+                with: ""
+            - map:
+                eroticspice: EroticSpice
+                forgivemefather: Forgive Me Father
+                loveherass: Love Her Ass
+                prettydirtyteens: Pretty Dirty Teens
+                sexworking: Sex Working
+      Image:
+        selector: //div[contains(@class,"player-poster")]/@style
+        postProcess:
+          - replace:
+            - regex: (.+url\(")(.+jpg)(.+)
+              with: $2
```

Comment:
this doesnt seem to work with some of the subsites
the script seems to work though
```yaml
      Image:
        selector: //script[@type="application/ld+json"]/text()
        postProcess:
          - replace:
            - regex: '.*"thumbnailUrl": "([^"]+)".*'
              with: $1
```